src/features/survey-forms/[id]/survey-form.tsx

```diff
@@ -0,0 +1,333 @@
+        price: 12,
+        image: getRandomImage(),
+      },
+      {
+        id: "c",
+        name: "Lobster",
+        description: "A large marine crustacean prized for its meat",
+        price: 14,
+        image: getRandomImage(),
+      },
+      {
+        id: "d",
+        name: "Crab",
+        description: "A crustacean with a broad, flat body and large pincers",
+        price: 18,
+        image: getRandomImage(),
+      },
+      {
+        id: "e",
+        name: "Sushi",
+        description:
+          "A Japanese dish consisting of vinegared rice combined with various ingredients, such as seafood and vegetables",
+        price: 16,
+        image: getRandomImage(),
+      },
+    ],
+  },
+  {
+    id: 4,
+    question: "What is your favorite dessert?",
+    choices: [
+      {
+        id: "a",
+        name: "Cake",
+        description: "A sweet baked dessert",
+        price: 20,
+        image: getRandomImage(),
+      },
+      {
+        id: "b",
+        name: "Ice Cream",
+        description: "A frozen dessert made from dairy products",
+        price: 25,
+        image: getRandomImage(),
+      },
+      {
+        id: "c",
+        name: "Cookies",
+        description: "Small, sweet baked treats",
+        price: 22,
+        image: getRandomImage(),
+      },
+      {
+        id: "d",
+        name: "Pie",
+        description: "A baked dish with a pastry crust filled with sweet or savory ingredients",
+        price: 21,
+        image: getRandomImage(),
+      },
+      {
+        id: "e",
+        name: "Pudding",
+        description: "A creamy dessert typically served chilled",
+        price: 24,
+        image: getRandomImage(),
+      },
+    ],
+  },
+  {
+    id: 5,
+    question: "What is your favorite type of cuisine?",
+    choices: [
+      {
+        id: "a",
+        name: "Mexican",
+        description: "Traditional dishes and flavors from Mexico",
+        price: 30,
+        image: getRandomImage(),
+      },
+      {
+        id: "b",
+        name: "Japanese",
+        description: "Cuisine from Japan, known for its sushi, ramen, and tempura",
+        price: 35,
+        image: getRandomImage(),
+      },
+      {
+        id: "c",
+        name: "Indian",
+        description:
+          "Flavorful dishes from India, often featuring spices and herbs asdsadadadas a asdasdsadsad asd asdasd asdasdasd asd asd asd as",
+        price: 32,
+        image: getRandomImage(),
+      },
+      {
+        id: "d",
+        name: "Thai",
+        description: "Spicy and aromatic cuisine from Thailand",
+        price: 33,
+        image: getRandomImage(),
+      },
+      {
+        id: "e",
+        name: "Italian",
+        description: "Cuisine from Italy, known for pasta, pizza, and risotto",
+        price: 34,
+        image: getRandomImage(),
+      },
+    ],
+  },
+]
+
+export default function SurveyForm() {
+  return (
+    <div>
+      <PageTitle>Test Survey Sample</PageTitle>
```

Comment:
this should be inside survey-form-header?

Reply:
Sure, I'll move it over there.